monolith/views/stories.py

```diff
@@ -10,8 +10,16 @@
 @stories.route('/stories')
 def _stories(message=''):
     allstories = db.session.query(Story)
-    return render_template("stories.html", message=message, stories=allstories, like_it_url="http://127.0.0.1:5000/stories/like/")
+    return render_template("stories.html", message=message, stories=allstories)
 
+@stories.route('/story/<story_id>')
+def _story(story_id, message=''):
+    story = Story.query.filter_by(id=story_id).first()
+    if story is None:
+        message = 'Story not found'
+    return render_template("story.html", message=message, story=story,
+                           like_it_url_="http::/127.0.0.1:5000/stories/like/",
+                           dislike_it_url_="http::/127.0.0.1:5000/stories/dislike/")
```

Comment:
The name of this two variables are different from the ones read by the html, leading to a pointeless pages.
Also you can remove the entire IP by adding the slash in front of the API you want to point. In this way the root is automatically added.
Concluding, what i'm suggesting is:
`return render_template("story.html", message=message, story=story,
                           like_it_url="/stories/like/",
                           dislike_it_url="/stories/dislike/")`

Reply:
Fixed.